### PR TITLE
PP-9796 OpenAPI specs for ChargeApiResource

### PR DESCRIPTION
**Pull request by kbottla**
## WHAT YOU DID
- Added annotations to generate OpenAPI specs for endpoints in ChargeApiResource which includes below.

  POST /v1/api/accounts/{accountId}/charges
  GET /v1/api/accounts/{accountId}/charges/{chargeId}
  POST /v1/api/accounts/{accountId}/telephone-charges
  GET /v1/api/charges/gateway_transaction/{gatewayTransactionId}
  POST /v1/tasks/expired-charges-sweep

- Preview using editor.swagger.io- 
image

### CODE REVIEW

File: src/main/java/uk/gov/pay/connector/charge/resource/ChargesApiResource.java [hunk 1]
```diff
@@ -110,8 +139,28 @@ public Response createNewCharge(
     @POST
     @Path("v1/api/accounts/{accountId}/telephone-charges")
     @Produces(APPLICATION_JSON)
+    @Operation(
+            summary = "Create a new telephone charge for gateway account.",
+            description = "Create a new telephone charge for gateway account. These are externally taken payments and the outcome is reported to this endpoint." +
+                    "provider_id is used as an idempotency key for API calls. If a payment already exists with the provider_id provided, the API will not store a record about a new payment, or update or change the record about a payment previously stored.",
```
Comment (SandorArpa) [inline src/main/java/uk/gov/pay/connector/charge/resource/ChargesApiResource.java:145]: Can we add space here, either at the end of previous sentence or at the beginning this sentence?

**Review by SandorArpa — CHANGES_REQUESTED** [review]

**Review by heathd — APPROVED** [review]
looks good

**Review by SandorArpa — APPROVED** [review]
LOKTM

File: src/main/java/uk/gov/pay/connector/charge/resource/ChargesApiResource.java (hunk 1 above)
Comment (kbottla) [reply src/main/java/uk/gov/pay/connector/charge/resource/ChargesApiResource.java:145]: Thanks! Will address in next PR

**Pull request merged**